tests/harness/cc/harness.cc

```diff
@@ -0,0 +1,78 @@
+#include <iostream>
+
+#include "tests/harness/cases/bool.pb.h"
+#include "tests/harness/cases/bytes.pb.h"
+#include "tests/harness/cases/enums.pb.h"
+#include "tests/harness/cases/maps.pb.h"
+#include "tests/harness/cases/messages.pb.h"
+#include "tests/harness/cases/numbers.pb.h"
+#include "tests/harness/cases/oneofs.pb.h"
+#include "tests/harness/cases/repeated.pb.h"
+#include "tests/harness/cases/strings.pb.h"
+#include "tests/harness/cases/wkt_any.pb.h"
+#include "tests/harness/cases/wkt_duration.pb.h"
+#include "tests/harness/cases/wkt_timestamp.pb.h"
+#include "tests/harness/cases/wkt_wrappers.pb.h"
+
+#include "tests/harness/harness.pb.h"
+
+using tests::harness::TestCase;
+using tests::harness::TestResult;
+using google::protobuf::Any;
+using google::protobuf::Message;
+
+std::ostream& operator<<(std::ostream& out, const TestResult& result) {
+  out << "valid: " << result.valid() << " reason: '" << result.reason() << "'"
+      << std::endl;
+  return out;
+}
+
+void resp(const TestResult& result) {
```

Comment:
Either add comments to functions explaining what they do, or make them totally obvious via the naming.